<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DateTimeSetupDialog</class>
 <widget class="QDialog" name="DateTimeSetupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DateTimeSetupDialog</string>
  </property>
  <widget class="QPushButton" name="pbOk">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>180</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCancel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>180</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbTime">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Time</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbFormat">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Format</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbDate">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Date</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbFormat">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>200</width>
     <height>35</height>
    </rect>
   </property>
  </widget>
  <widget class="DateLineEdit" name="leDate">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>70</y>
     <width>200</width>
     <height>35</height>
    </rect>
   </property>
  </widget>
  <widget class="TimeLineEdit" name="leTime">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>120</y>
     <width>200</width>
     <height>35</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>lineedit.h</header>
  </customwidget>
  <customwidget>
   <class>DateLineEdit</class>
   <extends>LineEdit</extends>
   <header>datelineedit.h</header>
  </customwidget>
  <customwidget>
   <class>TimeLineEdit</class>
   <extends>LineEdit</extends>
   <header>timelineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
